src/core/transaction.zig

```diff
@@ -75,7 +75,50 @@ pub const Transaction = struct {
         };
     }
 
-    pub const InitOwnedMsgWithSigningKeypairsError = error{
+    /// Basic transaction with randomized addresses and data.
+    ///
+    /// The number of instructions, pubkeys, and program/writable/signer/readonly
+    /// indexes are hardcoded, not randomized.
+    pub fn initRandom(allocator: std.mem.Allocator, random: std.Random) !Transaction {
+        const KeyPair = std.crypto.sign.Ed25519.KeyPair;
+        const keypair = try KeyPair.generateDeterministic(.{random.int(u8)} ** 32);
```

Comment:
Just to clarify: the intention is to generate a 32 byte array of one single homogeneous, randomly selected value?

Reply:
Yes that was the intention. This array is going to be used as a seed for the prng. The private key itself will become a different, non-homogeneous array.

This means our sample set of keypairs has a size of 256, which is quite small. Should be fine for tests, which is the only place where it would make sense to call this function. I'm not opposed to changing this to increase the size of the sample set. Should just be two more lines of code.

Reply:
I think that's fine, just wanted to clarify, thank you for elaborating.